__tests__/data/iterable_router_output.json

```diff
@@ -1,86 +1,43 @@
 [
   {
+    "batched": true,
     "batchedRequest": {
       "body": {
         "XML": {},
         "JSON_ARRAY": {},
         "FORM": {},
         "JSON": {
-          "userId": "abcdeeeeeeeexxxx102",
-          "createdAt": 1598631966468,
-          "eventName": "Email Opened",
-          "dataFields": {
-            "subject": "resume validate",
-            "sendtime": "2020-01-01",
-            "sendlocation": "akashdeep@gmail.com"
-          }
+          "batch": [
```

Comment:
It is supposed to be "events". Please verify the request body which Iterable expects.

Reply:
Done